src/test/java/com/iota/iri/service/tipselection/impl/WalkValidatorImplTest.java

```diff
@@ -38,6 +40,7 @@
     public static void tearDown() throws Exception {
         tangle.shutdown();
         dbFolder.delete();
+        WalkValidatorImpl.failedBelowMaxDepthCache.clear();
```

Comment:
This will run only once per class invocation, as `tearDown` is `@AfterClass`.
is that intentional? I don't see a reason to clear the cache after disposing of the DB.

If you want to run after each test:
```
@After
    public void tearDownEach() {
        WalkValidatorImpl.failedBelowMaxDepthCache.clear();
    }
```
* tests still pass if you add this.

Reply:
yes